### DISCUSSION TITLE
How to add a custom element to media Element Controls?

### DISCUSSION THREAD

**Discussion by ne0rrmatrix** [CommunityToolkit/Maui#1492] [Q&A]
I am looking to modify the xaml for media elements transport controls. I want to add a button for full screen. I am looking specifically on how to customize the control template. I want to put a full screen icon beside the aspect ratio button and tie it to a custom control I have.

**Comment by CliffAgius** [comment]
Closed as per the Community Stand-up discussion here: https://youtu.be/N9wMcBP4jtg?t=2889 where it was agreed that New Features need to get above 7 upvotes within 6 months to progress to being reviewed by the maintainers.
